Source/carthage/CopyFrameworks.swift

```diff
@@ -47,7 +47,7 @@ public struct CopyFrameworksCommand: CommandType {
 private func codeSigningIdentity() -> SignalProducer<String?, CarthageError> {
 	return SignalProducer.try {
 		if codeSigningAllowed() {
-			return getEnvironmentVariable("EXPANDED_CODE_SIGN_IDENTITY").map { $0 }
+			return getEnvironmentVariable("EXPANDED_CODE_SIGN_IDENTITY").map { $0.isEmpty ? nil : $0 }
```

Comment:
Can you explain this? I noticed that Xcode was not setting this environment variable for me in one of my targets.
What would the behavior be with this change?